third_party/web_platform_tests/service-workers/service-worker/registration-service-worker-attributes.https.html

```diff
@@ -65,8 +65,10 @@
           // immediately followed by setting active to null, which means by the
           // time the event loop turns and the Promise for statechange is
           // resolved, this will be gone.
-          assert_equals(registration.active, null,
-                        'active should be null after redundant');
+          // For Cobalt, this is not the case. Setting active to null can happen
+          // a little bit later after the Update State change.
+          // assert_equals(registration.active, null,
```

Comment:
Oh, I think this means that the statechange event can be sent too early. Maybe it goes away if you move the `DispatchEvent(statechange)` below the `RemoveServiceWorker`.

Reply:
Or, actually the missing thing is probably that the active worker on the registration is changed too late relative to the `statechange` event for redundant.

Reply:
Yep, this one is because when "DispatchEvent(statechange)" happens, UpdateRegistrationState() is not done yet. It's not required by spec as well. Do you think we need to fix this?